src/content/visualise/constants/Parameters.tsx

```diff
@@ -0,0 +1,118 @@
+export const intrinsicParameters = [
+  'phase',
+  'total_mass',
+  'mass_1',
+  'mass_2',
+  'symmetric_mass_ratio',
+  'spin_1x',
+  'spin_1y',
+  'spin_1z',
+  'spin_2x',
+  'spin_2y',
+  'spin_2z',
+  'phi_1',
+  'phi_2',
+  'chi_eff',
+  'chi_1_in_plane',
+  'chi_2_in_plane',
+  'chi_p',
+  'cos_tilt_1',
+  'cos_tilt_2',
+  'mass_1_source',
+  'mass_2_source',
+  'chirp_mass_source',
+  'total_mass_source'
+]
+export const extrinsicParameters = [
+  'azimuth',
+  'zenith',
+  'H1_time',
+  'luminosity_distance',
+  'geocent_time',
+  'dec',
+  'ra',
+  'iota',
+  'redshift',
+  'comoving_distance'
+]
+export const miscellaneousParameters = [
```

Comment:
Maybe change this to 'otherParameters'. Unless miscellaneous is actually the name for them, but 'other' is what the PMs discussed and will probably be the case on the uploading page